ymir/backend/src/common/proto/backend.proto

```diff
@@ -275,10 +275,12 @@ message Label {
 }
 
 message EvaluateConfig {
-    float iou_threshold_from = 1;
-    float iou_threshold_to = 2;
-    float iou_threshold_step = 3;
-    float conf_threshold = 4;
+    // confidence threshold, 0 to 1
+    float conf_thr = 1;
+    // from:to:step
```

Comment:
from:to:step, to value is excluded, same as python range

Reply:
updated